### ISSUE TITLE
Softly download latest version on version mismatch

### ISSUE THREAD

**Issue by paulschmeida** [SergeyPirogov/webdriver_manager#293]
Today Google released Chrome 98.0.4758.81, but the latest Chromedriver version is 98.0.4758.48, which is compatible with this release. I propose webdriver_manager should download latest compatible version if there's a version mismatch, not just when the Chrome version is unknown.

**Comment by paulschmeida** [comment]
I also just found that that latest release according to this: https://chromedriver.storage.googleapis.com/LATEST_RELEASE
is 97.0.4692.71, which is not true, because the current stable Chrome version is 98.0.4758.81 and current stable chromedriver version is 98.0.4758.48.

**Comment by aleksandr-kotlyar** [comment]
That's already should work like you describe. If not - please provide the webdriver_manager version and example of code to reproduce.

**Comment by paulschmeida** [comment]
Oh, in that case I think it doesn't work like that when the version of Chrome browser is supplied manually. I have a different way of retrieving current Chrome browser version, which uses wind32api. I had to do it because I package my projects using pyinstaller and webdriver_manager uses subprocess to get chrome version, which is incompatible with pyinstaller and other packagers that create .exe out of python projects.

I could see if I can change the way webdriver_manager looks for Chrome version on Windows to use win32api, if you guys are interested?

This is a more reliable way that subprocess:

`def _get_chrome_version(filename):
    try:
        info = win32api.GetFileVersionInfo(filename, '\\')
        ms = info['FileVersionMS']
        ls = info['FileVersionLS']
        return f'{win32api.HIWORD(ms)}.{win32api.LOWORD(ms)}.{win32api.HIWORD(ls)}.{win32api.LOWORD(ls)}'
    except:
        return 'Unknown version'`

**Comment by aleksandr-kotlyar** [comment]
If you send 98.0.4758.81 to webdriver-manager as version, and chrome doesn't have a webdriver for it - so it fails - it's okay.
You should take 98 from 98.0.4858.81 and send 98 version to webdriver-manager. Than it will download right version of chromedriver.

**Comment by paulschmeida** [comment]
I've tested both the release and the github version and it doesn't work like that. It tries to download this:
ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/98/chromedriver_win32.zip

and it fails.

Looking at the code, I don't believe that's how it webdriver-manager works. You need to supply the full version string and it needs to match and existing chromedriver version.

**Comment by aleksandr-kotlyar** [comment]
Provide code snippet how do you pass chromeversion to webdriver_manager.

**Comment by paulschmeida** [comment]
So I get the Chrome version using the above function and I then initialize webdriver_manager with a `version=` argument:

```
 # Chrome webdriver
        chrome_filename = r'C:\Program Files (x86)\Google\Chrome\Application\chrome.exe'
        driver = webdriver.Chrome(ChromeDriverManager(version=_get_chrome_version(chrome_filename)).install(), chrome_options=options)
```

`_get_chrome_version` returns a full version string like '98.0.4858.81'.

**Comment by aleksandr-kotlyar** [comment]
Even with your function - there is no way to get chromedriver version if chrome doesn't publish it.
You have '98.0.4858.81' version but Chrome published chromedriver with '98.0.4758.80'. win32api will not help you to get exact version of chromedriver in this case because it was not published.

**Comment by aleksandr-kotlyar** [comment]
The problem that should be solved is the more powerfull windows version reader than webdriver_manager has now. win32api seems to be a huge dependency which I will not recommend to include to this library. But maintainer can have another opinion. We need just one function from this library.

**Comment by paulschmeida** [comment]
> Even with your function - there is no way to get chromedriver version if chrome doesn't publish it. You have '98.0.4858.81' version but Chrome published chromedriver with '98.0.4758.80'. win32api will not help you in this case because the version with .81 was not published.

No, my code gets the latest published version for major browser version, so in this example it will get chromedriver '98.0.4758.80' for chrome browser '98.0.4858.81'. I already have it running in production.
